specification/resources/resource-manager/Microsoft.Resources/stable/2022-12-01/subscriptions.json

```diff
@@ -0,0 +1,959 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "SubscriptionClient",
+    "version": "2022-12-01",
+    "description": "All resource groups and resources exist within subscriptions. These operation enable you get information about your subscriptions and tenants. A tenant is a dedicated instance of Azure Active Directory (Azure AD) for your organization."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/providers/Microsoft.Resources/operations": {
```

Comment:
>operations [](http://example.com/codeflow?start=36&length=10)

Can u fix the errors flagged here - [https://github.com/Azure/azure-rest-api-specs/pull/23292/checks?check_run_id=12554205043](https://github.com/Azure/azure-rest-api-specs/pull/23292/checks?check_run_id=12554205043) ?

Reply:
Fixed